### ISSUE TITLE
internalFetch message no longer works according to Docmentation in Bob 1.2.1

### ISSUE THREAD

**Issue by mklauber** [OokTech/TW5-Bob#77]
According to the documentation a button like 

```
<$button>
  Import Some Tiddlers
  <$action-websocketmessage
    $type='internalFetch'
    fromWiki='AnotherWiki'
    filter='[[$:/SiteTitle]]'
    transformFilter='[addprefix[From Another Wiki: ]]'
  />
</$button>
```

should create a new tiddler titled `From Another Wiki: $:/SiteTitle`.  However, instead of doing so, it generates an error message in the nodejs console: `Couldn't parse externalTiddlers input` 

---

I am running (check any that apply, put an x inside the [ ] to check a box, like this: [x]):

- [ ] Windows
- [ ] OSX
- [x] Linux
- [ ] Other

and using

- [x] The nodejs version
- [x] The single file executable

Before posting I read [issue guidelines](./IssueGuidelines.md) and:

- [x] I am using the newest version
- [x] The answer to my question isn't listed in the documentation or this isn't
  a question
- [x] This is not a duplicate issue
- [x] I have not done anything that required me to set acceptance to
  `I Will Not Get Tech Support For This`

**Comment by inmysocks** [comment]
This should be fixed in the next release (1.2.2)

**Comment by mklauber** [comment]
Can confirm it's fixed in 1.2.2.  

FWIW, if you try to internal fetch a shadow tiddler from another wiki, it repeatedly tries and fails.  Not a huge issue for me, but letting you know.

**Comment by mklauber** [comment]
Thanks for all your hard work, and your incredible tool.
